services/QuillLMS/spec/controllers/admin_report_filter_selections_controller_spec.rb

```diff
@@ -76,6 +78,27 @@
           .and change { admin_report_filter_selection.reload.filter_selections['schools'][0]['name'] }.from(nil).to(school['name'])
       end
 
+      it "updates the filter selections for growth diagnostic dropdown filters" do
+        report = AdminReportFilterSelection::DIAGNOSTIC_GROWTH_REPORT_SKILL
+        grade_option = { 'value' => 'grade', 'label' => 'grade' }
+        classroom_option = { 'value' => 'classroom', 'label' => 'classroom' }
+        starter_diagnostic_option = { 'value' => '1663', 'label' => 'Starter Diagnostic' }
+        ell_starter_diagnostic_option = { 'value' => '1161', 'label' => 'ELL Starter Diagnostic' }
+
+        admin_report_filter_selection = create(:admin_report_filter_selection, user: user, report: report, filter_selections: { 'group_by_value' => grade_option, 'diagnostic_type_value' => starter_diagnostic_option })
+
+        updated_filter_selections = {
+          'group_by_value' => classroom_option,
+          'diagnostic_type_value' => ell_starter_diagnostic_option
+        }
+        updated_attributes = { report: report, filter_selections: updated_filter_selections }
+
+        expect do
+          post :create_or_update, params: { admin_report_filter_selection: updated_attributes }
+        end.to change { admin_report_filter_selection.reload.filter_selections['group_by_value']['value'] }.from('grade').to('classroom')
+          .and change { admin_report_filter_selection.reload.filter_selections['diagnostic_type_value']['value'] }.from('1663').to('1161')
```

Comment:
I'd rather see the variable names than their literal values in the `from` and `to` calls here.  That way we won't break this by accident if we update values somewhere else.